### PR TITLE
Figure S7

### PR DESCRIPTION
**Pull request by MikeLippincott**
Add Supplemental Figure 7

### CODE REVIEW

File: figures/S7.supplemental_figure7/scripts/S7.r [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+metadata_file_SHSY5Y <- metadata_file_SHSY5Y %>%
+    group_by(Metadata_Well, Metadata_cell_type, label) %>%
+    summarize(
+        n = n()
+    )
+head(metadata_file_SHSY5Y)
+
+
+# combine the SHSY5Y and PBMC metadata files
+combined_metadata <- rbind(
```
Comment (d33bs) [inline figures/S7.supplemental_figure7/scripts/S7.r:158]: By the time `metadata_file_PBMC` and `metadata_file_SHSY5Y` arrive here, what datatype are they? Would it make sense to use something besides `rbind` to combine them (for instance, [combine via `c()`](https://arrow.apache.org/docs/r/articles/dataset.html#work-with-multiple-data-sources))?

File: figures/S7.supplemental_figure7/scripts/S7.r [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    x = platemap_file,
+    y = combined_metadata,
+    by.x = "well_id",
+    by.y = "Metadata_Well",
+    all.x = TRUE
+)
+head(updated_platemap)
+unique(updated_platemap$Metadata_cell_type)
+# replace "" in cell_type with "Blank" in label
+# updated_platemap$Metadata_cell_type[updated_platemap$Metadata_cell_type == NA] <- "Blank"
```
Comment (d33bs) [inline figures/S7.supplemental_figure7/scripts/S7.r:197]: If this is no longer needed, consider removing.

File: figures/S7.supplemental_figure7/scripts/S7.r [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+options(repr.plot.width = width, repr.plot.height = height, units = "cm")
+# set pallette
+viridis_pal_custom <- viridis::viridis_pal(option = "C")(3)
+
+data_split_plate_map <- (
+    raw_map(
+        data = updated_platemap$label,
+        well = updated_platemap$well_id,
+        plate = 384,
+        size = 14)
```
Comment (d33bs) [inline figures/S7.supplemental_figure7/scripts/S7.r:242]: Consider documenting why this plate and size number (if that hasn't already been done elsewhere).

File: figures/S7.supplemental_figure7/scripts/S7.r [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+        'Treatment Holdout' = viridis_pal_custom[3]
+
+    )
+    )
+    + theme(
+        axis.text.x = element_text(size = 16),
+        axis.text.y = element_text(size = 16)
+    )
+
+)
```
Comment (d33bs) [inline figures/S7.supplemental_figure7/scripts/S7.r:278]: Consider adding a few comments to help describe what each section of this does and why.

**Review by d33bs — APPROVED** [review]
Nice job! I left a few comments and overall felt this looked good. Please don't hesitate to let me know if you have any questions.

File: figures/S7.supplemental_figure7/notebooks/S7.ipynb
[the remark this answers is not in the document]
Comment (MikeLippincott) [reply figures/S7.supplemental_figure7/notebooks/S7.ipynb:1]: addressed

File: figures/S7.supplemental_figure7/scripts/S7.r [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+    "../",
+    "../",
+    "4.sc_Morphology_Neural_Network_MLP_Model/0.hyperparameter_optimization/indexes/PBMC/multi_class/MultiClass_MLP_data_split_indexes.tsv"
+)
+index_file_path_SHSY5Y <- file.path(
+    "../",
+    "../",
+    "../",
+    "4.sc_Morphology_Neural_Network_MLP_Model/0.hyperparameter_optimization/indexes/SHSY5Y/multi_class/MultiClass_MLP_data_split_indexes.tsv"
+)
```
[the remark this answers is not in the document]
Comment (MikeLippincott) [reply figures/S7.supplemental_figure7/scripts/S7.r:23]: addressed and added

File: figures/S7.supplemental_figure7/scripts/S7.r (hunk 1 above)
Comment (MikeLippincott) [reply figures/S7.supplemental_figure7/scripts/S7.r:158]: They are data frames so rbind makes the most sense to me

File: figures/S7.supplemental_figure7/scripts/S7.r (hunk 2 above)
Comment (MikeLippincott) [reply figures/S7.supplemental_figure7/scripts/S7.r:197]: removed

File: figures/S7.supplemental_figure7/scripts/S7.r (hunk 3 above)
Comment (MikeLippincott) [reply figures/S7.supplemental_figure7/scripts/S7.r:242]: addressed

**Pull request merged**